### DISCUSSION TITLE
ASDF by default via git clone

### DISCUSSION THREAD

**Discussion by qbart** [Selleo/commons#10] [Ideas]
Most of us use .asdf:
- this would help with the automatic setup
- new comers and interns could just inherit script that prepares dev environment

It will only install if `~/.asdf` is not present.

**Comment by k1eu** [comment]
maybe also installing LTS versions of node/ruby and path config so the os doesn't use eg. Homebrew installed one?

**Reply by qbart** [reply]
I thought about using some default plugins but each environment is different,
let's try to keep things to minimum and then expand with extra features.

One idea would be to introduce "optional" dependencies or "presets" for example instead of doing:

```
./setup.sh
```
you would do:

```
PRESET=ruby ./setup.sh # or something like this: ASDF_DEFAULTS=true etc.
```

Wdyt?

**Reply by k1eu** [reply]
I like the ASDF_DEFAULTS approach
